src/backend/TrafficCourts/Citizen.Service/Controllers/TicketsController.cs

```diff
@@ -75,10 +76,14 @@ public TicketsController(IMediator mediator, ILogger<TicketsController> logger)
         [ProducesResponseType(StatusCodes.Status400BadRequest)]
         [RequestSizeLimit(10485760)]
         public async Task<IActionResult> AnalyseAsync(
-            [Required][PermittedFileContentType(new string[] { "image/png", "image/jpeg", "application/pdf" })] IFormFile file,
+            [Required]
+            //[PermittedFileContentType(new string[] { "image/png", "image/jpeg", "application/pdf" })] 

```

Comment:
I needed to temporarily comment out this line in order to get the OpenAPI spec to generate correctly. Even though I tried to upload a File of type png, this restriction made the generated client unable to use this endpoint.